squad/core/tasks/notification.py

```diff
@@ -15,6 +15,10 @@ def maybe_notify_project_status(status_id):
     build = projectstatus.build
     project = build.project
 
+    timeout = project.notification_timeout
+    if timeout:
+        notification_timeout.apply_async(args=[status_id], countdown=timeout)
```

Comment:
This guarantees a notification but only if maybe_notify_project_status has been called, but doesn't that require at least one result? What if no results have been received?

Same question about concurrency - could this result in duplicate emails if multiple notification_timeouts are queued up for the same moment?